contrib/fd-plugins/bareos_tasks/pgsql/README.md

```diff
@@ -64,8 +64,18 @@ Command (with or without full path) to *pg_dump* tool. Default: *pg_dump*
 #### pg_dump_options
 Options to be used with the *pg_dump* tool. Default: *--no-owner --no-acl*
 
+#### pg_host
+Specifies the host name of the machine on which the server is running.
+If the value begins with a slash,
+it is used as the directory for the Unix-domain socket.
+
+#### pg_port
+Specifies the TCP port
+or the local Unix-domain socket file extension
+on which the server is listening for connections.
+
 #### pg_user
-Username of the system user running the *psql* and *pg_dump* tools. Default: *postgres*
+Username of the system user running the *psql* and *pg_dump* tools. If an empty string is specified (`pg_user=''`) by the same user also running the  *bareos-fd*. Default: *postgres*
```

Comment:
Im not completely happy by how that sounds.  What do you think of the following ?

Specifies the user that will be used to run the *psql* and *pg_dump* tools.  If this option is set to the empty string (`pg_user=''`), then the user running *bareos-fd* is used.  Default: *postgres*

Reply:
I even changed the wording a bit more, based on your sentence.